orttraining/orttraining/python/training/ort_triton/_sympy_utils.py

```diff
@@ -9,6 +9,12 @@
 import sympy
 
 
+def extract_shape_from_symbol(symbol: str) -> int:
```

Comment:
Is this specific to some pattern? it the pattern fixed?

Reply:
Right. Symbol only creates for symbolic dims with this specific pattern (in triton_op_executor.py). Put this func here because there are multiple places call it.